package.json

```diff
@@ -2,36 +2,33 @@
   "name": "@github/textarea-autosize",
   "description": "Autosizes textarea to size of it's contents.",
   "version": "0.1.3",
-  "main": "dist/textarea-autosize.umd.js",
-  "module": "dist/textarea-autosize.esm.js",
-  "types": "index.d.ts",
+  "main": "dist/textarea-autosize.js",
+  "module": "dist/textarea-autosize.js",
+  "types": "dist/textarea-autosize.d.ts",
   "license": "MIT",
   "repository": "github/textarea-autosize",
   "files": [
-    "dist",
-    "index.d.ts"
+    "dist"
   ],
   "scripts": {
     "clean": "rm -rf dist",
     "lint": "github-lint",
     "prebuild": "npm run clean && npm run lint && mkdir dist",
-    "build": "rollup -c && npm run build-flow",
-    "build-flow": "cp textarea-autosize.js.flow dist/textarea-autosize.esm.js.flow && cp textarea-autosize.js.flow dist/textarea-autosize.umd.js.flow",
+    "build": "tsc && cp textarea-autosize.js.flow dist/textarea-autosize.js.flow",
```

Comment:
Let's remove Flow while doing this migration to TypeScript. We can add the Flow types inside github/github while we're still using it there.